postgres-demo/README.md

```diff
@@ -5,15 +5,15 @@ web application using Apiary and [Spring Boot](https://spring.io/projects/spring
 We use Apiary's Postgres backend.
 To get started, let's first install some dependencies:
 
-    sudo apt install openjdk-11-jdk maven libatomic1
+    sudo apt install openjdk-11-jdk maven libatomic1 postgresql-client
```

Comment:
If we could access psql through Docker, then we don't need to install `postgresql-client`.